src/rules/order-alphabetical.js

```diff
@@ -14,11 +14,8 @@ CSSLint.addRule({
     init: function(parser, reporter){
         "use strict";
         var rule = this,
-            properties;
-
-        var startRule = function () {
-            properties = [];
-        };
+            properties = [],
+            startRule = function() {};
```

Comment:
I don't think this change is correct…
Specifically, it causes the following test case to fail the rule, and I don't think it should fail.

    li { box-shadow: none; color: red; transition: none; }
    p { box-shadow: none; color: red; transition: none; }